cli/cmd/plugin/unmanaged-cluster/cluster/cluster.go

```diff
@@ -39,7 +39,8 @@ type Manager interface {
 	// PreflightCheck performs any pre-checks that can find issues up front that
 	// would cause problems for cluster creation. Returns nil if there are no
 	// errors found, otherwise a list of the errors that need to be resolved.
```

Comment:
Like on the next line? :) Or something else?

Reply:
> Returns nil if there are no errors found, otherwise a list of the errors that need to be resolved.

makes it sound like there's only a list of errors that gets returned 

but just a nit :) feel free to ignore!